hr_contract_multi_job/readme/INSTALL.rst

```diff
@@ -0,0 +1,7 @@
+To install this module, you need to:
```

Comment:
This install instruction is common for all modules, there's no value including in everywhere.